netebpfext/net_ebpf_ext_bind.c

```diff
@@ -388,15 +390,17 @@ net_ebpf_ext_resource_release_classify(
         goto Exit;
     }
 
-    attached_client = (net_ebpf_extension_hook_client_t*)filter_context->client_context;
-    if (attached_client == NULL) {
+    if (filter_context->client_detached) {
+        NET_EBPF_EXT_LOG_MESSAGE_NTSTATUS(
+            NET_EBPF_EXT_TRACELOG_LEVEL_VERBOSE,
+            NET_EBPF_EXT_TRACELOG_KEYWORD_SOCK_ADDR,
+            "Client has detached.",
```

Comment:
I left some comments above about this trace. Furthermore, can we differentiate between this and the above trace? Perhaps with the function name added to the trace? (Unless it is already captured in the macro).